SI/Sem.04/Pract.04/Task03.cpp

```diff
@@ -0,0 +1,36 @@
+using namespace std;
+#include <iostream>
+
+bool isPositive(int num)
+{
+    return num > 0 ? true : false;
```

Comment:
num > 0 is a boolean expression, so it returns a boolean value that you can directly return
return num > 0